### PR TITLE
Hardlinks to module files have been removed and the modulefiles directory has been created

### LINKED ISSUE #113: GODAS fails to run because of the necessary modules have been moved.

### PR DESCRIPTION
**Pull request by flampouris**
**Description**
0.  Load the updated module files. 
1. Removes the hard links for  "module load" from the JJOB scripts
2. Creates a modulefiles directory which will be populated as the system is ported to more HPCs, now it works only on hera.
3. The README.md has been updated. 

**Issue(s) addressed**
Closes #113 #112 

**Testing**

Run the system.

### CODE REVIEW

**Review by guillaumevernieres — CHANGES_REQUESTED** [review]
The README needs to be updated.

**Comment by guillaumevernieres** [comment]
Also, the sym link godas.main should eventually be done during an install step ... Something to keep in mind.

File: jobs/JJOB_DA_PREP [hunk 1]
```diff
@@ -11,9 +11,8 @@ cat <<EOF
 EOF
 
 module purge
-module use -a /scratch1/NCEPDEV/stmp4/Daniel.Holdaway/opt/modulefiles
-module load apps/jedi/intel-19.0.5.281
-module load nco
+source $MOD_PATH/godas.main
+moudule list
```
Comment (guillaumevernieres) [inline jobs/JJOB_DA_PREP]: typo: moudule should be module

File: jobs/JJOB_DA_3DVAR [hunk 2]
```diff
@@ -9,9 +9,10 @@ cat <<EOF
 #================================================================================
 #================================================================================
 EOF
+
 module purge
-module use -a /scratch1/NCEPDEV/stmp4/Daniel.Holdaway/opt/modulefiles
-module load apps/jedi/intel-19.0.5.281
+source $MOD_PATH/godas.main
+moudule list
```
Comment (guillaumevernieres) [inline jobs/JJOB_DA_3DVAR]: typo

File: jobs/JJOB_DA_PREP (hunk 1 above)
Comment (guillaumevernieres) [inline jobs/JJOB_DA_PREP]: typo

File: jobs/JJOB_GEN_ENSEMBLE [hunk 3]
```diff
@@ -6,8 +6,8 @@
 # Generate the ensemble
 
 module purge
-module use -a /scratch1/NCEPDEV/stmp4/Daniel.Holdaway/opt/modulefiles
-module load apps/jedi/intel-19.0.5.281
+source $MOD_PATH/godas.main
+moudule list
```
Comment (guillaumevernieres) [inline jobs/JJOB_GEN_ENSEMBLE]: typo

**Comment by flampouris** [comment]
Reviewer's comments have been addressed.

**Review by guillaumevernieres — APPROVED** [review]
Tested 3dvar only, extrapolating that envar and letkf are in the same status as in develop.

**Pull request merged**